dingdongg/spotify-payment
# M1 payment history page

**Description**

- created a payment history page for admin use
- payment table and filters refactored into their own component for future use

**To Test**

1. checkout this branch
2. `npm run dev`
3. navigate to "Admin Payment History" page
4. Check that the table is rendered correctly and scrolls properly
5. Check that the filters work as intended
